kv/kv.go

```diff
@@ -101,6 +110,9 @@ type Retriever interface {
 	// Get gets the value for key k from kv store.
 	// If corresponding kv pair does not exist, it returns nil and ErrNotExist.
 	Get(ctx context.Context, k Key) ([]byte, error)
+	// Get gets the value for key k from memory buffer only.
+	// If corresponding kv pair does not exist, it returns nil and ErrNotExist.
+	GetFromTxnMem(ctx context.Context, key Key) ([]byte, error)
```

Comment:
It's too expensive to add this method for `Retriever`.

Reply:
Done.